src/Features/Core/Portable/CodeFixes/Async/AbstractAddAwaitCodeFixProvider.cs

```diff
@@ -42,7 +43,7 @@ internal abstract partial class AbstractAddAwaitCodeFixProvider : AbstractAsyncC
         protected static bool TryGetTaskType(SemanticModel semanticModel, out INamedTypeSymbol taskType)
         {
             var compilation = semanticModel.Compilation;
-            taskType = compilation.GetTypeByMetadataName("System.Threading.Tasks.Task");
+            taskType = compilation.TaskType();
```

Comment:
Correct.  I had a dirty file still in VS from another refactoring unfortunately.